transformation/src/jade-schema/structs/xref.struct.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "xref",
```

Comment:
This structure is specialized to ClinVar, does it make sense to replicate here? As opposed to using the URL to the ENCODE website.

Reply:
Good catch. It sounds like we should just be able to store the xrefs as strings for encode.